3/debian-10/rootfs/opt/bitnami/scripts/libzookeeper.sh

```diff
@@ -166,10 +166,20 @@ zookeeper_validate() {
 
     # ZooKeeper server list validations
     if [[ -n $ZOO_SERVERS ]]; then
+        idFlag=false
+        if [[ "$ZOO_SERVERS" == *"::"* ]]; then
+            idFlag=true
+        fi
         read -r -a zookeeper_servers_list <<< "${ZOO_SERVERS//[;, ]/ }"
         for server in "${zookeeper_servers_list[@]}"; do
-            if ! echo "$server" | grep -q -E "^[^:]+:[^:]+:[^:]+$"; then
-                print_validation_error "Zookeeper server ${server} should follow the next syntax: host:port:port. Example: zookeeper:2888:3888"
+            if [[ "${idFlag}" == "false" ]]; then
```

Comment:
To be consistent with my previous suggestion:
```suggestion
            if is_boolean_yes "$server_id_with_jumps"; then
```